app/Http/Resources/FinalBillingProjectionResource.php

```
@@ -0,0 +1,24 @@
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'year_month' => $this['year_month'] ?? null,
+            'net_evenpar' => $this['net_evenpar'] ?? null,
+            'monthly_net_gt' => $this['net_evenpar'] ?? 0,
+            'projects' => ProjectsBillingSumarryResource::collection($this['projects']),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\ProjectsBillingSumarryResource' in method 'toArray'. (undefined)

(StaticAccess)